packages/size-limit/test/create-reporter.test.js

```diff
@@ -50,7 +53,9 @@ it('renders results', () => {
           time: 3,
           passed: true
         }
-      ]
+      ],
+      cleanDir: true,
+      saveBundle: join(tmpdir(), `size-limit-test`)
```

Comment:
I’ve just realised `createReporter` doesn’t do anything to filesystem, so I don’t need to make this test so complex.